src/components/data-pane/index.tsx

```diff
@@ -17,15 +17,15 @@ export class DataPanelBase extends React.PureComponent<DataPanelProps, {}> {
   public render() {
     const {name} = this.props.data;
     const fieldCount = this.props.data.schema.fieldSchemas.length;
-    const {showDataSourceSelector} = this.props.config;
+    const {showDataSourceSelector, manualSpecificationOnly} = this.props.config;
 
     const fields = fieldCount > 0 ? (
       <div styleName="data-pane-section">
         <h3>Fields</h3>
         <FieldList/>
       </div>) : null;
 
-    const wildcardFields = fieldCount > 0 ? (
+    const wildcardFields = !manualSpecificationOnly && fieldCount > 0 ? (
```

Comment:
I also wonder if we really needed fieldCount in here but that's beyond the scope of this PR.